scripts/india_aqi/preprocess.py

```diff
@@ -14,110 +14,119 @@
 
 import os
 import pandas as pd
-import numpy as np
-import urllib.request
 from datetime import datetime
 from os import path
-import requests, zipfile, io
+import zipfile
 
 module_dir = os.path.dirname(__file__)
+
+# AQI_STATIONS_FILE has list of Air Quality stations along with its DCIDs
 AQI_STATIONS_FILE = "India_AQI-stations_dcids_lgd.csv"
 
 
 class AQIDataLoader:
-    """ This class is responsible for loading and processing the unclean data into a clean by a series of steps
+    """
+    This class is responsible for loading and processing the unclean data into a clean by a series of steps
     and then finally appending it into the result CSV file.
 
     Instance Attributes:
-    source: The path of the CSV file whose data is later on processed and stored in the final CSV
-    raw_df: Unclean AQI pandas dataframe that undergoes preprocessing in order to become a clean dataframe
-    clean_df: Cleaned dataframe that has properly named columns, and compatible data types as per the column
+    source: The path of the CSV file whose data is later on processed and 
+            stored in the final CSV
+    raw_df: Unclean AQI pandas dataframe that undergoes preprocessing in order 
```

Comment:
nit: could we say "original" instead of "unclean"?

Reply:
Changed the word to 'original'.